community/csv/src/main/java/org/neo4j/csv/reader/BufferedCharSeeker.java

```diff
@@ -194,15 +194,15 @@ else if ( eof )
 
     private boolean setMark( Mark mark, int endOffset, int skippedChars, int ch, boolean isQuoted )
     {
-        int pos = (trim ? rtrim( bufferPos ) : bufferPos) - endOffset - skippedChars;
+        int pos = (trim ? rtrim() : bufferPos) - endOffset - skippedChars;
         mark.set( seekStartPos, pos, ch, isQuoted );
         return true;
     }
 
-    private int rtrim( int start )
+    private int rtrim()
```

Comment:
This whole method could use some more comments. There are some, but for the most part, especially with the new addition, it is black magic.